libraries/ESP8266WiFiMesh/examples/HelloMesh/HelloMesh.ino

```diff
@@ -1,50 +1,129 @@
 #include <ESP8266WiFi.h>
 #include <ESP8266WiFiMesh.h>
 
+String mesh_name = "Mesh_Node";
+
 unsigned int request_i = 0;
 unsigned int response_i = 0;
 
-String manageRequest(String request);
+String manageRequest(String request, ESP8266WiFiMesh *mesh_instance);
```

Comment:
The request argument is declared for passing by value, which needlessly duplicated mem. It should probably be declared as reference, or better a const reference.
While strictly speaking it's ok to declare the mesh_instance as a pointer, it's better to declare as reference to make it explicit that the arg can never be a nullptr.

Reply:
Makes sense.